content/CORETutorials/JailsPluginsVMs/Plugins/MinIOPlugin.md

```diff
@@ -0,0 +1,208 @@
+Click <i class="fa fa-stop" aria-hidden="true" title="Stop"></i>&nbsp;**STOP** to stop the jail before making any changes.
+
+Click <span class="material-icons">device_hub</span>&nbsp;**MOUNT POINTS** and follow the instructions in [Setting Up Jail Storage]({{< relref "SettingUpJailStorage.md" >}}) to mount the destination dataset you created in [First Steps](#first-steps).
+
+Click <span class="material-icons">play_arrow</span>&nbsp;**START** to restart the plugin and then click <span class="material-icons">settings</span>&nbsp;**MANAGE** to go to the **MinIO Console** and log in.
+
+{{< trueimage src="images/CORE/13.0/MinioPluginConsole.png" alt="MinIO Console" id="MinIO Console" >}}
+
+## Migrating from S3 Service to Minio Plugin
+
+After completing the initial install of the Minio plugin, users with an existing S3 service deployment need to migrate S3 data from the previous service deployment to the new plugin.
+Migrating from the built-in S3 service to the Minio plugin could require an extended service window.
+
+{{< enterprise >}}
+Beginning in CORE 13.0-U6, Enterprise customers with the S3 service running or enabled are prevented from upgrading to the next major version.
+
+TrueNAS Enterprise customers are strongly encouraged to contact iXsystems Support for assistance with the migration process.
+
+{{< expand "Contacting Support" "v" >}}
+{{< include file="content/_includes/iXsystemsSupportContact.md" >}}
+{{< /expand >}}
+{{< /enterprise >}}
+
+### Migration Overview
+Users with the S3 service enabled must upgrade CORE to version **13.0-U6** and complete the migration process **before** upgrading to a later major version.
+
+The migration path involves [installing a MinIO Client](#installing-the-minio-client) (`mc`) release with the required feature and function support to migrate the S3 service deployment.
+
+Next [configure the Minio plugin](#configuring-the-minio-plugin-for-migration) deployment and then [migrate data](#migrating-data) from the MinIO S3 service deployment to it.
+
+After migrating data, you can either create an archive of the TrueNAS S3 service or delete it.
+
+### Installing the MinIO Client
+
+The [MinIO Client](https://min.io/docs/minio/linux/reference/minio-mc.html) (`mc`) is command line tool with support for both filesystems and S3-compatible cloud storage services.
+Install `mc` to facilitate data the migration.
+Versions are available for Windows, Linux, and Mac.
+
+This tutorial uses the Windows Subsystem for Linux (WSL), but any standard Linux distribution follows the same process.
+Users should be familiar with WSL or the standard Linux CLI interface.
+
+{{< hint type=important >}}
+Installed MinIO client software version must be between **RELEASE.2022-06-26T18-51-48Z** and **RELEASE.2022-10-29T10-09-23Z**.
+Newer or older versions will not work properly.
+{{< /hint >}}
+
+To install the most recent compatible version of `mc`, enter:
```

Comment:
again, this is not a command in this instance it is a program/application/client.
```suggestion
To install the most recent compatible version of  MC, enter:
```